pkg/tfbridge/provider.go

```diff
@@ -674,16 +674,15 @@ func (p *Provider) Diff(ctx context.Context, req *pulumirpc.DiffRequest) (*pulum
 
 	// To figure out if we have a replacement, perform the diff and then look for RequiresNew flags.
 	olds, err := plugin.UnmarshalProperties(req.GetOlds(),
-		plugin.MarshalOptions{Label: fmt.Sprintf("%s.olds", label), KeepUnknowns: false})
```

Comment:
Any specific reason we not SkipNulls and no longer KeepUnknowns?

Reply:
The only addition here is `SkipNulls: true`; `KeepUnkonwns` is implicitly set to `false`. The addition of `SkipNulls` brings this call into alignment with other call sites that use the result to build TF state.